### DISCUSSION TITLE
您的网站效果太漂亮了！大佬有时间的时候可以写一篇关于部署到web的教程嘛～

### DISCUSSION THREAD

**Discussion by mattwong05** [oldwinter/knowledge-garden#4] [提问]
大佬好！

你的这个 [数字花园](https://oldwinter.top/AboutTheGarden) 真的太漂亮了。

我现在也有一个服务器 & 域名，也一直在用 Obsidian 作为自己的笔记工具。

我也想像您一样分享自己的 仓库，想请问您是如何部署的啊～ 

是购买了 Obsidian 的Publisher 吗？但是我看您的比官方的要漂亮好多，也没有最下面的标注。

我在 V2EX 上也看到了[您的分享](https://v2ex.com/t/857756) 但这里您分享的 https://notes.oldwinter.top/数字花园 使用 [Jekyll](https://github.com/oldwinter/dg)发布，但私以为这个没有您现在[主站](https://oldwinter.top/AboutTheGarden) 的好看，非常想知道您主站是如何部署的啊！

我翻看了您的笔记但是我没找到相关的内容，如果您方便的话，可以指点一下吗？

万分感谢🙏

**Comment by mattwong05** [comment]
不好意思，我没看仔细，找到了一篇您的[笔记](https://oldwinter.top/Calendar/已发布文章/obsidian+目前最完美的免费发布方案+-+渐进式教程)，正在研究呐～

**Reply by oldwinter** [reply]
不好意思，我没用过discussion这个功能，都不知道你在这里留言了。。导致现在才回复。。。
主站是要花钱的，obsidian的官方发布服务，而且是20美元/1个月。😂
